src/lsp/cobol_indent/indent_config.ml

```diff
@@ -62,9 +62,11 @@ let () =
       "SECTION", 0 ]
 
 let set_config ~indent_config =
-  let str = Ez_file.V1.EzFile.read_file indent_config in
-  let strlist = String.split_on_char '\n' str in
-  build_table strlist offset_table
+  match Ez_file.V1.EzFile.read_file indent_config with
```

Comment:
Please note: this should be temporary, and the config for the indenter should come as a pre-filled record for the `cobol_indent` library (typically, `indent_config.ml`, bound as `Cobol_indent.Config`, would only define the type of this record).

Reply:
Ah, this is #46 — I understand that issue now. Yeah, this is just a hotfix to be able to test stuff in spite of the hard-coded path used in `superbol indent`.